XamarinCommunityToolkitSample/ViewModels/Converters/DoubleToIntConverterViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Sample.ViewModels.Converters
+{
+	public class DoubleToIntConverterViewModel : BaseViewModel
+	{
+		double input;
+
+		public double Input
+		{
+			get => input;
+			set
+				{
+					input = Convert.ToDouble(value);
```

Comment:
but **value** is already double, why do we cast it again? :)

Reply:
that was an piece of code before creating the converters